Sanghyun/services/postService.js

```diff
@@ -0,0 +1,71 @@
+const postDao = require("../models/postDao");
+const userDao = require("../models/userDao");
+
+const createPost = async ({ title, content, userId, postingImageUrl }) => {
+  await postDao.createPost({ title, content, userId, postingImageUrl });
+  return "postCreated";
+};
+
+const getPostings = async () => {
+  const postings = await postDao.postingList();
+  return postings;
```

Comment:
```suggestion
  return postDao.postingList();
```
불필요한 변수 선언 및 동기처리 없이 한 줄로 작성할 수 있는 코드입니다.